Fantom/fantom-memecoins/src/mappings/mappingHandlers.ts

```diff
@@ -0,0 +1,64 @@
+import { Approval, Transfer, Address } from "../types";
+import {
+  ApproveTransaction,
+  TransferLog,
+} from "../types/abi-interfaces/Erc20Abi";
+import assert from "assert";
+import { Erc20Abi__factory } from "../types/contracts";
+import { contractAddress } from "../const";
+
+const erc20 = Erc20Abi__factory.connect(contractAddress, api);
+
+async function getOrCreateAddress(accountAddress: string): Promise<Address> {
+  let address = await Address.get(accountAddress);
+
+  if (address == undefined) {
+    address = Address.create({
+      id: accountAddress,
+      balance: (await erc20.balanceOf(accountAddress)).toBigInt(),
+    });
+  }
+
+  address.save();
+  return address;
+}
+
+export async function handleLog(log: TransferLog): Promise<void> {
+  logger.info(`New transfer transaction log at block ${log.blockNumber}`);
+  assert(log.args, "No log.args");
+
+  let from = await getOrCreateAddress(log.args.from);
+  let to = await getOrCreateAddress(log.args.to);
+  let value = log.args.value.toBigInt();
+
+  const transfer = Transfer.create({
+    id: log.transactionHash + "-" + log.logIndex,
+    blockHeight: BigInt(log.blockNumber),
+    toId: to.id,
+    fromId: from.id,
+    value: value,
+    contractAddress: log.address,
+  });
+
+  from.balance = BigInt(from.balance) - value;
+  to.balance = BigInt(to.balance) + value;
```

Comment:
I think it's safer to do a full `await erc20.balanceOf(accountAddress)` There might be other functions and activities that change this so best to clean reset.

Use Promise.all([..,..] for performance for these two queries